bedrock/mozorg/templates/mozorg/home/home-a.html

```diff
@@ -0,0 +1,280 @@
+          <p class="section-cta"><a href="{{ url('firefox.family.index') }}">{{ _('Discover more') }}</a></p>
+        </div>
+      </div>
+    </div>{#--/.mission-control --#}
+  </section>
+
+  <section class="section section-externals" id="externals">
+    <ul class="content">
+      <li>
+        <a class="add-ons" rel="external" href="https://addons.mozilla.org/">
+          <h3>{{ _('Add-ons') }}</h3>
+          <p>{{ _('Compare prices, check the weather, listen to music, send a tweet and more right from Firefox.') }}</p>
+        </a>
+      </li>
+      <li>
+        <a class="careers" rel="external" href="https://careers.mozilla.org/">
+          <h3>{{ _('Careers') }}</h3>
+          <p>{{ _('Learn about the benefits of working at Mozilla and view open positions around the world.') }}</p>
+        </a>
+      </li>
+      <li>
+        <a class="help" rel="external" href="https://support.mozilla.org/">
+          <h3>{{ _('Need help?') }}</h3>
+          <p>{{ _('Get answers to your questions about Firefox and all Mozilla products from our support team.') }}</p>
+        </a>
+      </li>
+    </ul>
+  </section>
+
+  <aside class="section section-newsletter" id="newsletter-subscribe">
+    <div class="content">
+    {% if LANG.startswith('en-') %}
+      {{ email_newsletter_form(newsletters='mozilla-foundation', title=_('Love the Web?'), subtitle=_('Get the Mozilla newsletter and help us keep it open and free.'), button_class='light') }}
+    {% else %}
+      {{ email_newsletter_form(button_class='light') }}
+    {% endif %}
+    </div>
+  </aside>
+
+</main>
+{% endblock %}
+
+{% block site_footer %}
+<footer class="section section-footer" id="footer">
+  <nav class="content">
+    <div class="footer-column">
+      <h1 class="logo"><a href="{{ url('mozorg.home') }}" data-link-type="footer" data-link-name="Mozilla">Mozilla</a></h1>
+      <p class="license">
+        {%- trans url=url('foundation.licensing.website-content') -%}
+        Portions of this content are ©1998–{{ current_year }} by individual
+        mozilla.org contributors. Content available under
+        a <a href="{{ url }}">Creative Commons license</a>.
+        {%- endtrans -%}
+      </p>
+    </div>
+
+    <div class="footer-column">
+      <ul class="links-join">
+        <li><a href="{{ url('mozorg.contact.contact-landing') }}" data-link-type="footer" data-link-name="Contact Us">{{ _('Contact Us') }}</a></li>
+        <li class="wrap"><a href="{{ donate_url('footer') }}" class="donate" data-link-type="footer" data-link-name="Donate">{{ _('Donate') }}</a></li>
+        <li class="clear"><a href="https://wiki.mozilla.org/Webdev/GetInvolved/mozilla.org" data-link-type="footer" data-link-name="Contribute to this site">{{ _('Contribute to this site') }}</a></li>
+      </ul>
+
+      {% if fx_footer_links %}
```

Comment:
This seems redundant here since we're not including any platform footer links. Can it be removed?